src/main/resources/assets/securitycraft/lang/es_es.json

```diff
@@ -0,0 +1,767 @@
+	"help.securitycraft.universal_block_reinforcer_lvl2.info": "El reforzador de bloques universal refuerza varios bloques de Minecraft y los convierte en la variante de SecurityCraft. El nivel 2 tiene 2700 usos. Golpea cualquier bloque (sin modo creativo) con el reforzador para reforzarlo.",
+	"help.securitycraft.universal_block_reinforcer_lvl3.info": "El reforzador de bloques universal refuerza varios bloques de Minecraft y los convierte en la variante de SecurityCraft. El nivel 3 tiene usos ilimitados. Golpea cualquier bloque (sin modo creativo) con el reforzador para reforzarlo.",
+	"help.securitycraft.briefcase.info": "El maletín es un almacenamiento portátil con contraseña con 12 espacios. Al fabricarlo, haz clic derecho para establecer una contraseña de 4 dígitos. Vuelve a hacer clic derecho para introducir la contraseña e interactura con los objetos. Para eliminar la contraseña, mételo en una mesa de trabajo con un cambiador de contraseñas universal.",
+	"help.securitycraft.universal_key_changer.info": "El cambiador de contraseñas universal te permite cambiar las contraseñas de tus propios bloques protegidos. Haz clic derecho en un bloque protegido con contraseña para abrir la interfaz y escribe una contraseña.",
+	"help.securitycraft.secret_sign_item.info": "El cartel secreto funciona igual que el cartel estándar, con la diferencia de que solo el propietario puede ver el texto que hay en él. ¡Asegúrate de reforzar el bloque sobre el que lo colocas!",
+	"help.securitycraft.sentry.info": "La torreta tiene 3 modos que puedes elegir haciendo clic derecho sobre ella. Apagada (no atacará), camuflada (solo atacará a los jugadores cercanos) y agresiva (atacará a todos los jugadores y criaturas hostiles). Haz clic derechos sobre ella con un módulo de disfraz que contenga un bloque opaco (como piedra, nunca cristal o escaleras) para camuflar la torreta por completo. El módulo se puede cambiar directamente or extraerse haciendo clic derecho con el modificador de bloques universal. Para eliminar la torreta haz clic derecho sobre ella mientras estás agachado.",
+	"help.securitycraft.crystal_quartz_item.info": "El cuarzo de cristal se utiliza para fabricar bloques de cuarzo de cristal.",
+
+	"gui.securitycraft:scManual.disabled": "Desactivado",
+	"gui.securitycraft:scManual.recipe.keycard_lv1": "Desactivado",
+	"gui.securitycraft:scManual.recipe.keycard_lv2": "Desactivado",
+	"gui.securitycraft:scManual.recipe.keycard_lv3": "Desactivado",
+	"gui.securitycraft:scManual.recipe.keycard_lv4": "Desactivado",
+	"gui.securitycraft:scManual.recipe.keycard_lv5": "Desactivado",
+	"gui.securitycraft:scManual.recipe.limited_use_keycard": "Desactivado",
+	"gui.securitycraft:scManual.intro.1": "Primeros pasos en SecurityCraft",
+	"gui.securitycraft:scManual.intro.2": "de Geforce y bl4ckscor3",
```

Comment:
You can translate add the following line below the one this comment is at, which will show your name as the translator in the SC Manual:
`"gui.securitycraft:scManual.author": "Translated by Ryo567",`

Reply:
Oh this is cool :D 

`"gui.securitycraft:scManual.author": "Traducido por Ryo567",`

Regards!
(must I modify the file branch? Or is it enough with this comment reply? I'm a bit noob at github haha)

Reply:
As said in a different comment, you need to actively edit the file (see which one in the aforementioned comment). GitHub doesn't magically know what to change.

Reply:
Okay! I'm doing it :D Thanks!